frontend/src/components/Event.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {
+  Card,
+  CardHeader,
+  CardMedia,
+  CardContent,
+  Typography,
+  Button,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import app from "../firebase";
+import "firebase/compat/firestore";
+import "firebase/compat/auth";
+
+const Event = (props) => {
+  const navigate = useNavigate();
+
+  const handleSignUp = () => {
+    const user = app.auth().currentUser;
```

Comment:
Might make sense to export the result of `auth` directly rather than having to call it everywhere. Similar to what [the Firebase auth example](https://firebase.google.com/docs/auth/web/start#add-initialize-sdk) does?